packages/block-editor/src/store/private-selectors.js

```diff
@@ -84,14 +84,22 @@ export const isBlockSubtreeDisabled = ( state, clientId ) => {
 function getEnabledClientIdsTreeUnmemoized( state, rootClientId ) {
 	const blockOrder = getBlockOrder( state, rootClientId );
 	const result = [];
-
 	for ( const clientId of blockOrder ) {
+		// @TODO create a custom selector for this to limit potential damage?
```

Comment:
Added to this selector for demonstration hack.

A new private selector?

Or rather process and memoize in the ListView component?